symbols/ram.yml

```diff
@@ -685,3 +685,10 @@ ram:
         This data is populated as the dungeon floor is generated.
         
         type: struct floor_generation_status
+    - name: STAIRS_MENU_PTR
+      address:
+        EU: 0x238AEC4
+      length:
+        NA: 0x4
```

Comment:
It seems weird that there's no EU length here. Is there a reason to omit it?

Reply:
Oops. I ctrl v'ed it from somewhere and didn't check.